packages/node/test/e2e/requester-fulfill.feature.ts

```diff
@@ -80,39 +80,38 @@ it('should call fail function on AirnodeRrp contract and emit FailedRequest if r
 
   await e2e.makeRequests(deployConfig, deployment);
 
-  const preinvokeLogs = await e2e.fetchAllLogs(provider, deployment.contracts.AirnodeRrp);
-
-  const preinvokeTemplateRequests = preinvokeLogs.filter((log) => log.name === 'MadeTemplateRequest');
-  const preinvokeFullRequests = preinvokeLogs.filter((log) => log.name === 'MadeFullRequest');
-  const preinvokeFulfillments = preinvokeLogs.filter((log) => log.name === 'FulfilledRequest');
-
-  expect(preinvokeLogs.length).toEqual(5);
-  expect(preinvokeTemplateRequests.length).toEqual(1);
-  expect(preinvokeFullRequests.length).toEqual(1);
-  expect(preinvokeFulfillments.length).toEqual(0);
+  const preInvokelogNames = (await e2e.fetchAllLogs(provider, deployment.contracts.AirnodeRrp)).map(({ name }) => name);
+  expect(preInvokelogNames).toEqual([
+    'SetAirnodeXpub',
+    'SetSponsorshipStatus',
+    'CreatedTemplate',
+    'MadeTemplateRequest',
+    'MadeFullRequest',
+  ]);
 
   const chain = e2e.buildChainConfig(deployment.contracts);
   const config = fixtures.buildConfig({ chains: [chain] });
   jest.spyOn(fs, 'readFileSync').mockReturnValue(JSON.stringify([config]));
 
   await handlers.startCoordinator();
 
-  const postinvokeLogs = await e2e.fetchAllLogs(provider, deployment.contracts.AirnodeRrp);
-
-  const postinvokeTemplateRequests = postinvokeLogs.filter((log) => log.name === 'MadeTemplateRequest');
-  const postinvokeFullRequests = postinvokeLogs.filter((log) => log.name === 'MadeFullRequest');
-  const postinvokeFulfillmentsFailed = postinvokeLogs.filter((log) => log.name === 'FailedRequest');
-  const postinvokeFulfillments = postinvokeLogs.filter((log) => log.name === 'FulfilledRequest');
-
-  expect(postinvokeLogs.length).toEqual(7);
-  expect(postinvokeTemplateRequests.length).toEqual(1);
-  expect(postinvokeFullRequests.length).toEqual(1);
-  expect(postinvokeFulfillmentsFailed.length).toEqual(1);
-  expect(postinvokeFulfillments.length).toEqual(1);
-
-  // Check each fulfillment failed is linked to a request
-  postinvokeFulfillmentsFailed.forEach((fulfillment) => {
-    const request = postinvokeLogs.find((log) => log.args.requestId === fulfillment.args.requestId);
-    expect(request).toBeDefined();
+  const postInvokeLogs = await e2e.fetchAllLogs(provider, deployment.contracts.AirnodeRrp);
+  expect(postInvokeLogs.map(({ name }) => name)).toEqual([
+    'SetAirnodeXpub',
+    'SetSponsorshipStatus',
+    'CreatedTemplate',
+    'MadeTemplateRequest',
+    'MadeFullRequest',
+    'FailedRequest',
+  ]);
```

Comment:
I believe here is still missing the check that `full` request made should have been fulfilled with this event `FulfilledRequest`